src/errors/test-run/index.js

```diff
@@ -411,3 +417,11 @@ export class WindowDimensionsOverflowError extends TestRunErrorBase {
         this.callsite = callsite;
     }
 }
+
+export class RoleSwitchInRoleInitializerError extends TestRunErrorBase {
```

Comment:
I think better to move this error (and `WindowDimensionsOverflowError ` too) above [this line](https://github.com/inikulin/testcafe/blob/ddb507d6ef72b46d87f78b692558cc265d25a94c/src/errors/test-run/index.js#L353) because another way the located in group of `// Native dialog errors`